### ISSUE TITLE
Mantra create -- nothing is happen

### ISSUE THREAD

**Issue by andreyantipov** [mantrajs/mantra-cli#42]
Hello

Faced with problem when trying to create a new mantra project

Nothing is happen. No log output is appear with verbose too

``` shell
mantra create -v Tes1
  create  Tes1/
  invoke  init
```

Just infinite process

_Node and meteor ver._

``` shell
meteor 1.2.1 global
node v0.12.4
```

Is it possible to create a new empty meteor project (version 1.2 branch) and then update it to 1.3 branch manually and then recreate structure by mantra?

```
meteor update --release 1.3-beta.16
```

smth. like `mantra create .` in root of meteor application.

**Comment by merlinstardust** [comment]
I've run into a similar issue. It doesn't happen every time I run it, but it happens often enough that's it an issue. I wish it was more reproducible.

It just hangs after logging `invoke init`. 

Versions

```
$ node --version
v4.4.0

$ npm --version
2.14.20

$ meteor --version
Meteor 1.2.1
```

Since no output from `meteor create` is being generated, it must be hanging on these lines

```
shelljs.set('-e');
let currentPath = shelljs.pwd();
```

I haven't used shelljs before so I don't know what the issue could be.

**Comment by itsdillon** [comment]
I'm getting the same issue. Stuck at the invoke init.

I tried creating a directory using "meteor create --release 1.3-beta.11" then ran "mantra create ." in the root but still nothing is happening.

**Comment by andreyantipov** [comment]
> I tried creating a directory using "meteor create --release 1.3-beta.11" then ran "mantra create ." in the root but still nothing is happening.

@merlinpatt, @dillonraphael As i understand, `mantra create [path]` can't rebuild version of current project. Perhaps this way it's better for mantra (for better compatibility) but if you didn't have (in this case 1.3-rc.1) specific release you need wait until it be downloaded and installed. And if you have slow internet connection like me, it can be very long, i installed Meteor 1.3-rc.1 manually and problems disappear. 
Also i think, there can be another problem with `shelljs.exec timeout process`, but not sure.

``` javascript
 logger.invoke('init');
    shelljs.set('-e');
    let currentPath = shelljs.pwd();
    shelljs.exec(`meteor create ${appPath} --release 1.3-rc.1`, {silent: !options.verbose});
    shelljs.cd(appPath);
    shelljs.rm('-rf', ['client', 'server']);
    `kadira:flow-router${lineBreak}`.toEnd('.meteor/packages');
    shelljs.cd(currentPath);
```

I'd like propose to inform user about that process can be running long, because verbose mode didn't catch meteor installation output.
Btw. 1.3-rc.3 is out!

``` javascript
    shelljs.exec(`meteor create ${appPath} --release 1.3-rc.1`, {silent: !options.verbose});
```

**Comment by itsdillon** [comment]
@AndreyAntipov Was an experiment to see if it would get around the issue. Regardless running "mantra create [path] just gets stuck on the init.

**Comment by andreyantipov** [comment]
I solve it installing 

``` shell
meteor create newApp --release 1.3-rc.1
```

before mantra installation and then create a new app by mantra as usual.

**Comment by merlinstardust** [comment]
To be clear, I'm not trying to create a meteor project first. But I didn't think about how it has to download the latest version first. That would make a lot of sense

**Comment by andreyantipov** [comment]
> To be clear, I'm not trying to create a meteor project first. But I didn't think about how it has to download the latest version first. That would make a lot of sense

I see.

Added a pull request #45 , with changes.
- fixed readme create section
- added verbose warning when meteor is installing
- updated release to rc3

**Comment by sungwoncho** [comment]
@AndreyAntipov is right. If the Meteor release the CLI is using is not found in your machine, `init` stage can take a long time and we need to inform users about that. We will have solved that problem by the time #45 is merged.
